setup.py

```diff
@@ -114,6 +114,7 @@ def get_extensions():
         extension(
             "torchao._C",
             sources,
+            py_limited_api=True,
```

Comment:
It's nice to have it, but imo not necessary. At least by default, linker will ignore the dependencies, unless one uses some symbols from that library (unless library is linked with `-Wno-as-needed`)

Reply:
This is necessary only to get python to recognize the extension + therefore the wheel as python agnostic.

Reply:
This is also for our cppextension to know not to drag in libtorch_python